src/SDKs/EventGrid/management/Management.EventGrid/Microsoft.Azure.Management.EventGrid.csproj

```diff
@@ -6,10 +6,10 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.EventGrid</PackageId>
     <Description>Provides developers with a library to create and manage all Azure EventGrid resources. </Description>
-    <Version>3.0.1-preview</Version>
+    <Version>4.0.0</Version>
     <AssemblyName>Microsoft.Azure.Management.EventGrid</AssemblyName>
     <PackageTags>Microsoft Azure EventGrid Management;Event Grid;Event Grid management;</PackageTags>
-    <PackageReleaseNotes>Taking dependency on 10.0.3 version of Newtonsoft nuget package.</PackageReleaseNotes>
+    <PackageReleaseNotes>Introducing 2019-01-01 as new GA API version.</PackageReleaseNotes>
```

Comment:
Suggest replacing this with "This corresponds to the 2019-01-01 stable API version that supports General Availability of the following functionality related to event subscriptions: DeadLetter destination, Storage queue as destination, HybridConnection as destination, manual handshake validation, and support for retry policies. Features that are still in preview (such as Event Grid domains or advanced filters support) can still be accessed using the 3.0.1-preview version of the SDK."

Reply:
fixed